packages/flame/analysis_options.yaml

```diff
@@ -94,7 +94,7 @@ linter:
     - prefer_is_not_empty
     - prefer_is_not_operator
     - prefer_iterable_whereType
-    - prefer_mixin
+#    - prefer_mixin
```

Comment:
Should we just remove that line in total? Or have a follow up PR where we clean this up(as we reference it in code as well I believe).

/cc @flame-engine/flame-colab

Reply:
What does it do? Prefer a mixin over what?

Reply:
Found it [here](https://dart-lang.github.io/linter/lints/prefer_mixin.html), I don't really get how the two cases that they show there can be comparable though.

Reply:
@wolfenrain In my projects, I prefer to comment-out linter rules (perhaps even with a comment why), as opposed to straight-out deleting them, because then it's clear which rules, were deliberately deleted, and which were never there to begin with. For example, if the linter gets updated with new rules, I can then compare the new list vs what I have in the project and add the difference. However, if you have different conventions in `flame`, then it's fine too.

@spydon In my case, the analyzer complained that I use `... with ChangeNotifier`, even though `ChangeNotifier` is declared as a `class` and not a `mixin`. However, the docs for `ChangeNotifier` state that it can be used both as a parent and as a mixin, so the rule was giving a false positive.

Reply:
I personally don't have any preference for it, makes sense to document certain rules that are turned off as well.

But I know we have removed rules in the past, that's why I added it. I think we should choose a convention and stick to it, and your explanation has gotten my vote haha

Reply:
Let's keep it like this for now and we can discuss it in the meeting later and possibly do a follow-up. :)